src/Components/Home/ContributionSection.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+import styles from './styles';
+import SectionHeader from '../SectionHeader';
+
+
+function ContributionSection() {
+    return (
+        <View>
+        <SectionHeader title= "INTERESTED IN COMPUTING" />
+        <Text style={styles.description}>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed</Text>

```

Comment:
Lorem ipsum will not be accepted.

Reply:
@annabauza Lorem ipsum was given in the design. I can change it but what should I change it to?

Reply:
@annabauza @RiaDhanani 'You can start contributing to AnitaB.org Open Source through one or more fields of your choice' hows this?

Reply:
All texts should be held in separate files. Grid should not be created entirely manually it should be 6 same components. Why repead code? On screanshots texts feel shifted not centred to the images. Also I can't tell from screashot how is responsiveness of this implementation.

Reply:
Is the content issue yet created for this section? If yes is the pr raised for it? @vj-codes